tasks/skylab/TagSortBam.wdl

```diff
@@ -72,19 +87,31 @@ task GeneSortBam {
 
   command {
     set -e
+    mkdir temp
+
+    TagSort --bam-input ~{bam_input} \
+    --metric-output gene-metrics.csv  --compute-metric \
+    --metric-type gene \
+    --gene-tag GX \
+    --barcode-tag CB \
+    --umi-tag UB \
+    --temp-folder temp \
+    --alignments-per-thread 1000000 \
+    --nthreads 6
+
+    gzip gene-metrics.csv
 
-    TagSortBam -i ${bam_input} -o gene-sorted.bam -t GE CB UB
   }
 
   runtime {
     docker: docker
     memory: "${machine_mem_mb} MiB"
-    disks: "local-disk ${disk} HDD"
+    disks: "local-disk ${disk} HDD" #TODO: make it an input
```

Comment:
still a TODO?

Reply:
removed it. I am not sure if we want the user to specify if they want SDD or HDD... HDD is sufficient